examples/use-cases/push_events_to_syslog/pull_events.py

```diff
@@ -0,0 +1,83 @@
+import automox_console_sdk as automox
+import os
+import time
+import socket
+import json
+
+from automox_console_sdk import EventsApi
+from getpass import getpass
+
+def getRecentEvents(event_id=None):
+    try:
+        page = 0
+
+        while True:
+            recent_events = []
+            events_page = events_api.get_events(o=organization, limit=500, page=page)
+
+            if len(events_page) == 0:
+                return recent_events
+
+            for event in events_page:
```

Comment:
Are we comfortable that new events will always come first in response?

Reply:
This works currently. I could compare against a timestamp... A timestamp would actually save me a call against the api in the end and probably would be a better solution than relying on consistency.

Reply:
Iterating through the whole data set for new events sucks though. Since we reliably get the data from the API back in descending order this **_should_** be fine.

Reply:
Works for me if we have confidence 👍